src/jdbc-auth.md

```diff
@@ -1435,59 +1405,49 @@ descriptive name with the protocol to be used for the connection. It contains
 the following columns:
 
 `connection_id`
-: The unique integer associated with each connection. This value is 
-  generated automatically when a new entry is inserted into the 
-  `guacamole_connection` table.
+: A unique integer for each connection, generated automatically upon creating a
+  new entry in the `guacamole_connection` table.
 
 `connection_name`
-: The unique name associated with each connection. This value must be 
-  specified manually, and must be different from any existing connection 
-  name in the same connection group. References to connections in other 
-  tables use the value from `connection_id`, not `connection_name`.
+: A unique name for each connection. This value must be manually specified and
+  unique within the same connection group. Other tables reference connections
+  using `connection_id`, not `connection_name`.
 
 `protocol`
-: The protocol to use with this connection. This is the name of the protocol
-  that should be sent to guacd when connecting, for example "`vnc`" or 
-  "`rdp`".
+: The protocol for the connection (e.g., "`vnc`" or "`rdp`").
 
 `parent_id`
 : The unique integer associated with the connection group containing this 
   connection, or `NULL` if this connection is within the root group.
 
 `max_connections`
-: The maximum number of concurrent connections to allow to this connection 
-  at any one time *regardless of user*. `NULL` will use the default value 
-  specified in `guacamole.properties` and a value of `0` denotes unlimited.
+: The maximum concurrent connections allowed to this connection. `NULL` uses the
+  default value from `guacamole.properties`, and `0` denotes unlimited.
 
 `max_connections_per_user`
-: The maximum number of concurrent connections to allow to this connection  
-  at any one time *from a single user*. `NULL` will use the default value  
-  specified in `guacamole.properties` and a value of `0` denotes unlimited.
+: The maximum concurrent connections allowed per user. `NULL` uses the default
```

Comment:
It's probably still worth being clear that the limit is for _this connection_ somewhere. It's unlikely that anyone's going to get confused about it, but still.